src/components/projects-summary/projects-summary.controller.ts

```diff
@@ -144,10 +184,33 @@ export class ProjectsSummaryController implements angular.IController {
     }
   }
 
+  public orderService(item: any) {
+    this.$scope.$emit('open-overlay-panel', item);
+  }
+
   public showAllProjects() {
     var cb: any = this.ctrl.showProjects();
     if (cb) {
       cb();
     }
   }
+
+  private getRecentlyViewedItems() {
+    // recentItems is an array of uids
+    let recentItems: any = this.RecentlyViewed.getItems();
+
+    // replace uids with IServiceItems
+    let items: any = _.map(recentItems, (recentItem: any) => {
+      return _.find(this.allItems, (item: any) => {
```

Comment:
Shouldn't need `_.find` since you already indexed the items by UID.

```ts
let items = _.map(recentItems, (uid: string) => {
  return this.allItems[uid];
};
```